release-notes/7.0/preview/api-diff/preview6/Microsoft.NETCore.App/7.0-preview6_System.Numerics.md

```diff
@@ -0,0 +1,136 @@
+# System.Numerics
```

Comment:
`System.Numerics` LGTM.

CC. @michaelgsharp, @dakersnar

Reply:
Should AllBitsSet show up somewhere in here?